platform/logging/appenders/base/BaseAppenderConfig.ts

```diff
@@ -0,0 +1,27 @@
+import * as schema from '../../../lib/schema';
+import { typeOfSchema } from "../../../types";
+import { BaseAppender } from './BaseAppender';
+
+export function getRawSchema() {
+  return {
+    kind: schema.string(),
+    pattern: schema.string({ defaultValue: '[{timestamp}][{level}][{context}] {message}'})
+  };
+}
+
+const schemaType = typeOfSchema(() => schema.object(getRawSchema()));
```

Comment:
I'm not sure if I like this piece of code (and similar one for sub-classes), but I don't see better way right now.